src/components/Page.js

```diff
@@ -0,0 +1,38 @@
+const Page = ({ page }) => {
```

Comment:
ohhhh, a "page" === a wikipedia entry

i still think page is a little too generic. maybe "entry", "location", or "listItem" would be more descriptive?

Reply:
Good point! ... thank you!. We changed to `location`:

![image](https://user-images.githubusercontent.com/8689897/126823033-d5299067-91da-40e9-971e-7a75eff80cfb.png)